content/doc/administration/requirements/windows.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: documentation
+title:  Windows Support Policy
+---
+
+This page documents the Windows support policy for the Jenkins server and agents.
+
+== Scope
+
+Jenkins plugins, e.g. plugin:windows-slaves[WMI Windows Agents],
+may set additional requirements to Windows versions on masters and/or agents.
+This page does not document such requirements,
+please refer to plugin documentation.
+
+== Why?
+
+Theoretically Jenkins can run everywhere where you can run a supported Java version,
+but there are some limitations in practice.
+Jenkins core and some plugins include native code or depend on Windows API and subsystems,
+and hence they rely on specific Windows platforms and versions.
+In the case of Windows services we also use link:https://github.com/winsw/winsw[Windows Service Wrapper (WinSW)] which requires .NET Framework.
+
+== Support levels
+
+We define multiple support levels for Windows platforms.
+
+[width="100%",cols="20%,40%,40%",options="header",]
+|===
+|Support level |Description |Platforms
+
+| **Level 1** - Full support
+| We run automated testing for these platforms, and we intend to timely fix the reported issues.
+a| * 64-bit versions of latest Windows and Windows Server versions,
```

Comment:
is this s overly broad.  which server version do we actively test with (it's only one is my guess not all the flavours)

Reply:
Good point, we do not really test it.